### PR TITLE
refactor(admob): delete unused snippets

### PR DESCRIPTION
**Pull request by rosariopfernandes**
The build is failing in #285 due to AdMob Breaking Changes.
I was going to upgrade this code to the [new admob v20.0.0](https://developers.google.com/admob/android/migration), but then I noticed that most of it is not even shown on [our docs](https://firebase.google.com/docs/admob/android/quick-start#set-up-app-in-admob). Maybe it's time to clean it up?

### CODE REVIEW

**Comment by samtstern** [comment]
@rosariopfernandes I think you're right, these are unused. Thanks for catching this!

**Comment by samtstern** [comment]
Oops this is my bad but two of these snippets are actually used:
https://firebase.google.com/docs/admob/android/quick-start#add-and-initialize-sdk

I'll have to restore those.

**Comment by rosariopfernandes** [comment]
@samtstern The link you posted takes me to this section:

Screenshot from 2021-04-15 19-25-11

If you're talking about the `MobileAds.initialize()` snippet, it was not removed by this PR. It's still in this repo.

Other that this one, I don't see any other snippet in that docs page you linked.

File: admob/app/src/main/java/com/google/firebase/example/admob/kotlin/MainActivity.kt [hunk 1]
```diff
@@ -1,125 +1,16 @@
...
-    private lateinit var loadInterstitialButton: Button
-    private val context = this
-
     // [START ads_on_create]
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         // ...
         MobileAds.initialize(this)
     }
     // [END ads_on_create]
```
Comment (rosariopfernandes) [inline admob/app/src/main/java/com/google/firebase/example/admob/kotlin/MainActivity.kt:15]: @samtstern Here's the snippet I kept.

**Comment by samtstern** [comment]
Yep you're right, it was a false alarm! Something else broke the page and I assumed it was this.

**Pull request merged**